### PR TITLE
DOCS-171 - add prerequisites for JDBC Source Connector quickstart

### PR DESCRIPTION
**Pull request by joel-hamill**
image

Staging server:
- http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/663/3.3.0/connect/connect-jdbc/docs/source_connector.html#quick-start
- http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/663/3.3.0/connect/connect-jdbc/docs/sink_connector.html#quick-start

### CODE REVIEW

File: docs/includes/prerequisites.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. Prerequisites for using JDBC connector
+
+Prerequisites
+   - If you are using the JDBC source connector with |CP|, you must :ref:`install and start Confluent Platform <installation>` before using.
+   - If you are using the JDBC source connector with a cluster running Apache Kafka, you must have you have ZooKeeper, Kafka, and Schema Registry running before using.
```
Comment (kkonstantine) [inline docs/includes/prerequisites.rst]: Do we distinguish in our CP docs between connector deployments on top of CP or just AK? I thought we always assumed users use Confluent platform.

Comment (joel-hamill) [reply docs/includes/prerequisites.rst]: @kkonstantine in general, i agree with you that we should should assume and recommend CP installation as the default path. i think i may have misunderstood this issue at first. so it sounds like someone just needs to rewrite this quickstart to use the confluent CLI (and not add verbiage about running with standalone AK)?

this recently came up re: C3 and using with standalone AK.  we solved by creating 2 separate installation paths: [with CP](https://docs.confluent.io/current/control-center/docs/installation/installing-control-center.html) and [with AK](https://docs.confluent.io/current/control-center/docs/installation/install-apache-kafka.html). however, C3 may be a unique case.

Comment (joel-hamill) [reply docs/includes/prerequisites.rst]: @kkonstantine how can i make this work with CP CLI and non-standalone Connect?

**Comment by joel-hamill** [comment]
@kkonstantine ready for final review: 

Staging server:

http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/663/3.3.0/connect/connect-jdbc/docs/source_connector.html#quick-start
http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/663/3.3.0/connect/connect-jdbc/docs/sink_connector.html#quick-start

File: docs/source_connector.rst [hunk 2]
```diff
@@ -11,92 +11,120 @@ The database is monitored for new or deleted tables and adapts automatically. Wh
...
 
-   sqlite> CREATE TABLE accounts(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, name VARCHAR(255));
-   sqlite> INSERT INTO accounts(name) VALUES('alice');
-   sqlite> INSERT INTO accounts(name) VALUES('bob');
+Load the predefined JDBC source connector.
 
-Now we create a configuration file that will load data from this database. This file is included
-with the connector in ``./etc/kafka-connect-jdbc/source-quickstart-sqlite.properties`` and contains the
-following settings::
+#.  Optional: View available connectors with this command:
```
Comment (kkonstantine) [inline docs/source_connector.rst]: If you change above, then please change here too. 

_View available predefined connectors with _

File: docs/source_connector.rst [hunk 3]
```diff
@@ -11,92 +11,120 @@ The database is monitored for new or deleted tables and adapts automatically. Wh
...
+.. sourcecode:: bash
 
-   {"id":3,"name":{"string":"cathy"}}
+    {"id":3,"name":{"string":"cathy"}}
 
-Note that the default polling interval is 5 seconds, so it may take a few seconds to show up.
-Depending on your expected rate up updates or desired latency, a smaller poll interval could be
-used to deliver updates more quickly.
+Note that the default polling interval is five seconds, so it may take a few seconds to show up. Depending on your expected
+rate up updates or desired latency, a smaller poll interval could be used to deliver updates more quickly.
```
Comment (kkonstantine) [inline docs/source_connector.rst]: _up updates_ typo?

File: docs/includes/prerequisites.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
+.. Prerequisites for using JDBC connector
+
+**Prerequisites:**
+
+- :ref:`Confluent Platform <installation>` is installed and running by using the Confluent CLI. This quick start assumes that you are using the Confluent CLI, but standalone installations are also supported. By default ZooKeeper, Kafka, Schema Registry, Kafka Connect REST API, and Kafka Connect are started with the ``confluent start`` command. For more information, see :ref:`installation_archive`.
```
Comment (kkonstantine) [inline docs/includes/prerequisites.rst]: How about: 
_Confluent Platform is installed. **Services are** running by using the Confluent CLI_?

to be extra clear that CP is **not** installed by CLI?
(not sure about the full stop between the two sentences - your call in general)

File: docs/source_connector.rst (hunk 3 above)
Comment (joel-hamill) [reply docs/source_connector.rst]: good catch

**Pull request merged**